data/MadhuSaini22.json

```diff
@@ -30,35 +41,52 @@
       "name": "Interested in Open Source stuff? follow me on Twitter",
       "url": "https://twitter.com/MadhuSaini22",
       "icon": "FaTwitter"
+    },
+    {
+      "name": "Blog - I write about Open Source and Full Stack Development",
+      "url": "https://hashnode.com/@MadhuSaini22",
+      "icon": "SiHashnode"
     }
   ],
   "milestones": [
     {
-      "title": "Made Chatting Application",
-
-      "date": "May 2021",
-      "icon": "code",
-      "color": "blue",
-      "description": "This is a fully responsive chat application with social authentication including Google and Facebook using Firebase online statuses, great design and functionality, image support, ability to create multiple rooms and much more.",
-      "url": "https://github.com/MadhuSaini22/Unichat"
+      "title": "Social Media",
+      "icon": "FaAndroid",
+      "color": "green",
+      "description": "Started becoming active on my Twitter account, I've had my Twitter account for past months, but didn't know Tech Twitter",
+      "url": "https://twitter.com/MadhuSaini22"
     },
-    {
-      "title": "Made Virtual Assistant",
-
-      "date": "April 2021",
-      "icon": "code",
-      "color": "red",
-      "description": "Virtual voice assistant with features like speak to search Google and Wikipedia, ask the time, play music and YouTube, open VS Code and Reddit, etc.",
-      "url": "https://github.com/MadhuSaini22/Virtual_Assistant"
+     {
+      "title": "Technologies I know",
+      "date": "2023",
+      "icon": "FaGlobe",
+      "color": "green",
+      "description": "HTML, CSS, JavaScript, Typescript, Tailwind, Express, Nodejs, React, Next, MongoDB",
+      "url": "https://madhusaini.netlify.app/"
     },
     {
-      "title": "Made Web Video Caller",
-
-      "date": "November 2021",
-      "icon": "code",
+      "title": "Became a Full Stack Developer",
+      "date": "2022",
+      "icon": "FaGraduationCap",
       "color": "green",
-      "description": "Video Chat Application, in which you can have fun with your best buddy by easy video chit chatting.",
-      "url": "https://github.com/MadhuSaini22/web-rtc-video-chat-app"
-    }
+      "description": "Currently working as a Full Stack Developer from Indian Startup",
+      "url": "https://enacton.com/"
+    },
+     {
+      "title": "Open-Source",
+      "date": "2022",
+      "icon": "SiOpensourceinitiative",
+      "color": "#000",
+      "description": "My open source journey started after joining EddieHub and 4C community",
+      "url": "https://github.com/MadhuSaini22"
+    },
+     {
+      "title": "Became Staff Member at 4C",
+      "date": "2022",
+      "icon": "CgCommunity",
```

Comment:
`CgCommunity` icon is not supported by LinkFree, the default globe icon will be placed here in this case. If you want, you can update it to a [supported icon](https://linkfree.eddiehub.io/icons).